### PR TITLE
examples/suit_update: some test fixes

### PR DESCRIPTION
**Pull request by kaspar030**
### Contribution description

Seems like this test wasn't updated when we introduced suit_workers.

### Testing procedure

The test-with-config make target fails on master, succeeds with this PR.

### CODE REVIEW

File: examples/suit_update/tests-with-config/01-run.py [hunk 1]
```diff
@@ -31,6 +31,8 @@
 IFACE = os.getenv("IFACE", "tapbr0")
 TMPDIR = tempfile.TemporaryDirectory()
 
+TRIGGER_RECEIVED_MSG = "suit_worker: started."
+REBOOTING_MSG = "suit_worker: rebooting..."
 
 def get_iface_addr(iface):
```
[the remark this answers is not in the document]
Comment (kaspar030) [reply examples/suit_update/tests-with-config/01-run.py:38]: yup

[the remark this answers is not in the document]
Comment (kaspar030) [reply examples/suit_update/tests-with-config/01-run.py:38]: squashed right in

**Review by benpicco — APPROVED** [review]

**Comment by aabadie** [comment]
bors merge

**Comment by aabadie** [comment]
bors cancel

**Pull request merged**